config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -66,30 +67,76 @@ presubmits:
         - "--timeout=420m"
         securityContext:
           privileged: true
 
+  # pull-cloud-provider-azure-e2e-ccm runs Azure specific e2e tests.
+  - name: pull-cloud-provider-azure-e2e-ccm
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-cloudprovider-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/kubernetes=v1.14.0-alpha.1"
```

Comment:
Does this need to be updated to match support for `v1.14.0-rc.1`?

Reply:
better to match, but a little patch version difference is unavailable since those are in different repos. Fortunately, it's still working with patch version differences.

Reply:
Let me update those versions to rc.1 instead.